packages/cli/test/dev/integration-1.test.ts

```diff
@@ -967,6 +967,7 @@ test(
   testFixtureStdio('handle-miss-querystring', async (testPath: any) => {
     await testPath(200, '/', 'Index Page');
     if (process.env.CI && process.platform === 'darwin') {
+      // eslint-disable-next-line no-console
```

Comment:
Is there a way to disable this rule for tests?

Reply:
@erikareads and I discussed this briefly but thought maybe we also want to clear some of that out. I think our options here are either:

- Ignore it at the top of the file, but that might be annoying to do
- Put tests in the `.eslintignore`, but that would ignore all eslint rules